Sources/TokamakDOM/Views/Text.swift

```diff
@@ -17,9 +17,106 @@ import TokamakCore
 
 public typealias Text = TokamakCore.Text
 
+extension Font.Design: CustomStringConvertible {
+  /// Some default font stacks for the various designs
+  public var description: String {
+    switch self {
+    case .default:
+      return "system, -apple-system, \".SFNSText-Regular\", \"San Francisco\", \"Roboto\", \"Segoe UI\", \"Helvetica Neue\", \"Lucida Grande\", sans-serif"
```

Comment:
I wonder if raw strings could make it more readable. This also could be the case where separate lines would make it more maintainable, as this bit is not as repetitive as `Font.swift`. 

```suggestion
      return #"""
      system,\
      -apple-system,\
      ".SFNSText-Regular",\
      "San Francisco",\
      "Roboto",\
      "Segoe UI",\
      "Helvetica Neue",\
      "Lucida Grande",\
      sans-serif
      """#
```